android/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<resources>
+  <string name="klaviyo_react_native_sdk_version">1.0.0</string>
```

Comment:
Nit: my suggestion would be to use a more generic name like 
`klaviyo_sdk_version_override`
`klaviyo_sdk_name_override`